tests/unit/test_charm.py

```diff
@@ -212,16 +215,14 @@ def test_given_unit_is_leader_and_ca_certificate_saved_when_collectstatus_then_s
 
         self.assertEqual(
             self.harness.charm.unit.status,
-            WaitingStatus("Waiting for CA certificate in workload"),
+            ActiveStatus(),
```

Comment:
This part should be given close review. I think it makes more sense to be like this -- but I don't want to be changing the tests to match the behaviour.

Reply:
The only thing I could see is that we are not explicit with the status of the service, in the test setup or title. I agree that the previous version was obviously wrong.